lib/pbench/cli/server/user_management.py

```diff
@@ -100,7 +100,7 @@ def user_delete(context: object, username: str) -> None:
     try:
         # Delete the the user with specified username
         User.delete(username=username)
-        click.echo(f"User {username} deleted")
+        click.echo(f"User {username} deleted", err=True)
```

Comment:
See? Doesn't labeling this `err=True` make you feel a bit nauseous? That's when you apply the rule that it'd be better to just delete it entirely. ;-)